src/components/RankingSection/ProductCard.tsx

```diff
@@ -1,19 +1,19 @@
 import styled from '@emotion/styled';
-import type { Product } from '@/types/product';
 import { useNavigate } from 'react-router-dom';
 import { useAuth } from '@/contexts/AuthContext';
 import { ROUTES } from '@/constants/routes';
+import type { Product } from '@/types/product';
 
 interface ProductCardProps {
-  item: Product;
+  product: Product;
   rank: number;
 }
 
-const ProductCard = ({ item, rank }: ProductCardProps) => {
+const ProductCard = ({ product, rank }: ProductCardProps) => {
```

Comment:
```suggestion
const ProductCard = ({rank, ...product }: ProductCardProps) => {
```
이렇게 표현할 수 있도록 구성해주면 어떨까요?